contracts/WidoStarknetRouter.sol

```diff
@@ -12,6 +12,7 @@ import "hardhat/console.sol";
 
 contract WidoStarknetRouter {
     using SafeTransferLib for ERC20;
+    using SafeTransferLib for address;
     
     IWidoConfig public widoConfig;
     IStarknetMessaging public starknetCore;
```

Comment:
if the contract stays like it is now, those two attributes would benefit from being `immutable`, it would save gas because instead of having _storage load_ instructions at runtime, it simply replaces the values on the bytecode.

Reply:
Same with `IWidoRouter public widoRouter` LOC#29